SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -191,7 +191,7 @@ internal static void AddCustomLanguageLine(string modAssemblyName, string lineId
 
         internal static string TrimTextDelimiters(string value)
         {
-            return value.Trim(TextDelimiterOpen, TextDelimiterClose);
+            return value.Substring(1, value.Length - 2);
```

Comment:
@AlexejheroYTB pinging you for discussion of ideas.  
Note that Unity uses the single line feed character '\n' in its text.
### Important Special Characters
```CSharp
char TextDelimiterOpen = '{';
char TextDelimiterClose = '}';
char KeyValueSeparator = ':';
```

There are two ways I can see to fully support `string.format` placeholders and line breaks.  

#### Option 1
Completely re-write the serialization format into something that can safely ignore the important special characters.  
**Pros**: Written strings will match what is being sent to SMLHelper.  
**Cons:** Windows might auto-fix line breaks. Requires a lot more coding.

#### Option 2
Replace the special characters with proxy characters when writing the original language files and reverse this process when reading from them.
**Pros**: Little code change needed. Allows one entry per line.
**Cons:** Multi-line entries will be on a single line, separate by some special character or sequence, which may be confusing.

In all cases, language files will likely need to be rebuilt.

Reply:
I would like to propose another option: escaping `{`, `}`, `:`, `\` with `\`, so they aren't treated as special characters by the parser

Reply:
Adding Escapes would require updating the parser to actually handle them in the first place.  
This isn't a very complex parser. This is all the important code from it:
```CSharp
string[] languageLines = File.ReadAllLines(file, Encoding.UTF8);

Dictionary<string, string> originalLines = originalCustomLines[modName];

int overridesApplied = 0;
for (int lineIndex = 0; lineIndex < languageLines.Length; lineIndex++)
{
    string line = languageLines[lineIndex];
    if (string.IsNullOrEmpty(line))
        continue; // Skip empty lines

    string[] split = line.Split(new[] { KeyValueSeparator }, StringSplitOptions.RemoveEmptyEntries);

    if (split.Length != 2)
        continue; // Not correctly formatter

    string key = split[0];
    if (!originalLines.ContainsKey(key))
        continue; // Skip keys we don't recognize.

    customLines[key] = TrimTextDelimiters(split[1]);
    overridesApplied++;
}
```